shacl12-core/index.html

```diff
@@ -1539,25 +1539,24 @@ <h3>Property Shapes</h3>
 					<span data-syntax-rule="path-maxCount">A shape has at most one <a>value</a> for <code>sh:path</code>.</span>
-					<br/><br/>
-					<span data-syntax-rule="path-node">The <a>value</a> of <code>sh:path</code> in a property shape is either a <a>well-formed</a>
-					<a>SHACL property path</a> or a <a>well-formed</a> <a>node expression</a>.</span>
-					<span data-syntax-rule="path-expr-order">The syntax rules for <a>SHACL property paths</a> take precedence over <a>node expressions</a>;
-					for example, this means that an <a>IRI</a> will be interpreted as a <a>predicate path</a> instead of an <a>IRI expression</a>.</span>
-					<span data-syntax-rule="path-not-literal">The <a>value</a> of <code>sh:path</code> cannot be a <a>literal</a>.</span>
+					<span data-syntax-rule="path-node">The <a>value</a> of <code>sh:path</code> in a property shape is a <a>well-formed</a>
+					<a>SHACL property path</a>.</span>
 					<br/><br/>
 					It is recommended, but not required, for a <a>property shape</a> to be declared as a <a>SHACL instance</a> of <code>sh:PropertyShape</code>.
 					<span data-syntax-rule="PropertyShape-path-minCount"><a>SHACL instances</a> of <code>sh:PropertyShape</code> have one <a>value</a> for the property <code>sh:path</code>.</span>
+					<br/><br/>
+					<span data-syntax-rule="path-values">A <a>property shape</a> has at most one <a>value</a> for the property <code>sh:values</code> and this <a>value</a> is a <a>well-formed</a> <a>node expression</a>.</span>
+					<span data-syntax-rule="path-defaultValue">A <a>property shape</a> has at most one <a>value</a> for the property <code>sh:defaultValue</code> and this <a>value</a> is a <a>well-formed</a> <a>node expression</a>.</span>
+					<span data-syntax-rule="path-values-iri">A <a>property shape</a> can only have a <a>value</a> for <code>sh:values</code> or <code>sh:defaultValue</code> when its <a>value</a> for <code>sh:path</code> is an <a>IRI</a>.</span>
 				</p>
 				<p>
 					Informally, property shapes specify constraints that need to be met with respect to <a>nodes</a> that can be reached from the
 					<a>focus node</a> either by directly following a given property (specified as an <a>IRI</a>) or any other <a>SHACL property path</a>,
-					specified using <code>sh:path</code>.
+					specified using <code>sh:path</code>, or evaluating the node expression specified using <code>sh:values</code>, or (if no other values exist)
+					by evaluating the node expression specified using <code>sh:defaultValue</code>.
 				</p>
 				<p>
-					If the <code>sh:path</code> is not a <a>well-formed</a> property path, it must be a <a>node expression</a>
-					and the constraints will be validated against the <a>output nodes</a> of this node expression.
-					Note that support for node expressions is not required by SHACL Core,
+					Note that support for <code>sh:values</code> and <code>sh:defaultValue</code> is not required by SHACL Core,
```

Comment:
For a SHACL Core engine, what should be its behaviour when it encounters a Property Shape with `sh:values` and `sh:path`?

Does it ignore `sh:values` and look for node by following the path?
Does it ignore the property altogether?
Does it fail?

Reply:
The idea here is that we don't want to break SHACL Core implementations. So we cannot expect them to even know about sh:values and therefore we cannot even expect them to fail. In other words, they can just look for the path values and ignore sh:values/defaultValue.

Reply:
Right, but then in practice it means that in cases of constraints which require existence of values (`sh:hasValue`, `sh:MinCount`) you will likely get false-negative validation result (no nodes at derived property). And skipping the property altogether implies a potential false-positive outcome for nodes which would otherwise fail validation if `sh:values` would have been resolved.

Maybe Core client should at least acknowledge that it saw and recognised a property it cannot process and add a `sh:Warning` result to the validation report? 

Or is it something to leave to implementations? One can think of many such cases when SHACL silently ignores problems, leading to subtle errors in downstream code

Reply:
Good question. The situation is similar to any new constraint component that 1.2 introduces. 1.0 engines would just ignore them and then probably produce incorrect or missing results. Maybe this discussion requires its own ticket.

Reply:
That should be covered by the profiles. Maybe we can use SHACL-SHACL to identify such problems. I opened an issue for [that topic](https://github.com/w3c/data-shapes/issues/312).